relay-server/src/actors/store.rs

```diff
@@ -1028,6 +1029,9 @@ struct CheckInKafkaMessage {
     #[serde(skip)]
     routing_key_hint: Option<Uuid>,
 
+    /// Discriminate tag used to differentiate these messages from clock pulse messages that are
+    /// also placed into the topic.
```

Comment:
Could we reword this to meet [relay coding guidelines (internal)](https://www.notion.so/sentry/Relay-Code-and-Architecture-4fac481cf27741aba54d39594f1cae58?pvs=4#987c0416a9f54674999eed59eecbf846)? Thanks!

> ensure that there is a precise first line, followed by more information

Reply:
I think I fixed this